content/applications/finance/documents.rst

```diff
@@ -91,23 +112,106 @@ be modified and it is possible to access the related :guilabel:`Journal Entry` o
      rights to the document. It is possible to replace the owner of a document. I.e.: an employee
      must be owner of a document to be able to see it in "My Profile".
 
-Finally, different :guilabel:`Actions` are available at the bottom of the right panel, depending on
-the workspace where your document is stored.
+Different :ref:`Actions <documents/workflow-actions>` are available at the bottom of the right
+panel, depending on the workspace where your document is stored.
+
+Split documents
+---------------
+
+Select the document you want to split, and click the :guilabel:`Split` icon. A new view opens and
+displays all the pages of the document.
+
+The scissors between pages indicate where the split will occur. Click on the scissors if you
+don't want to split your document between two specific pages. Click :guilabel:`Split` to split your
+document.
+
+.. image:: documents/split-pdf.png
+   :align: center
+   :alt: split your documents
+
+.. note::
+   To merge documents, select them from your dashboard and click the :guilabel:`Split` icon. Click
+   on the scissors between pages to remove the cut, and click :guilabel:`Split` to merge the
+   documents.
+
+Additional features
+-------------------
+
+Select a workspace and click on the arrow next to the :guilabel:`Upload` button to access additional
+features:
+
+Request
+~~~~~~~
+
+You can create "missing documents" and organize them as documents, to remind users to download these
+documents.
+
+Select the workspace in which the file should be stored and click :guilabel:`Upload`, then
+:guilabel:`Request`. Add the :guilabel:`Document name`, select the person you need it from in the
+:guilabel:`Request to` field, fill in the :guilabel:`Due date`, confirm the :guilabel:`Workspace`,
+the document should belong to, and add :guilabel:`Tags` and a :guilabel:`Note`, if needed. Then,
+click :guilabel:`Request`. A file reflecting the missing document is created in the workspace.
+
+You can see all missing documents by going to the :guilabel:`Activity` view and to the
+:guilabel:`Requested Document` column.
+
+.. Note::
+
+   To send a **reminder email** to all users you are waiting a document from, go to the
+   :guilabel:`Activity` viewclick on ⋮, and :guilabel:`Document Request: Reminder`.
```

Comment:
I commented before to specify more about where to find activities but I think you didn't add it.

Reply:
Updated. I think it's clear enough. Let me know if it's still not according to you.